.github/workflows/test.yml

```diff
@@ -0,0 +1,27 @@
+name: Unit tests CI
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [16.x]
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v3
+      with:
+        node-version: ${{ matrix.node-version }}
+        cache: 'yarn'
+    - name: Install dependencies
+      run: yarn bootstrap
```

Comment:
Before running `yarn bootstrap`, please consider running `yarn` first to install the `learna` version specify in `package.json` instead of relying on to `npx` install whatever it want.
![ee](https://user-images.githubusercontent.com/11730168/218692407-a85dda10-1ec6-48d3-88b0-4935d7956415.PNG)

Reply:
resolved